lib/Controller/LoginController.php

```diff
@@ -328,6 +332,27 @@ private function login($uid, Profile $profile)
             }
         }
 
+        # Create groups via the provider
+        $group_ids = [];
+        foreach($profile->data->groups as $provider_group) {
```

Comment:
If no `data->groups` in response will be warning

Reply:
Where will there be a warning? I didn't found a warning and everything worked fine even without data->groups.

Reply:
Check logs. At least will be something like `Invalid argument supplied for foreach()` plus `Trying to get property 'groups' of non-object` if no `data` in response etc.